<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器-黄展鸿-2022414120109</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/zhanh/Desktop/ppt背景/OIP-C (1).jpg</normaloff>C:/Users/zhanh/Desktop/ppt背景/OIP-C (1).jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	font-size:12pt;
	border-radius:2px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="btnClearAll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(255, 255, 127);
	color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(42, 42, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="btnPeriod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnDaoShu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>1/x</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="btnNum3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnPercentage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnNum8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnNum4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="btnPOM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnNum2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btnSqrt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>√x</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnSquare">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>x²</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnNum5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btnNum6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnClear">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(255, 255, 127);
	color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(42, 42, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="btnNum0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnNum7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnNum9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btnNum1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:hover{
	color: rgb(255,255,255);
	background-color: rgb(0, 0, 127);
}

QPushButton{
	color: rgb(0, 0, 0);
	background-color: rgb(85, 255, 255);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}


</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btnDel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(255, 255, 127);
	color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(42, 42, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>⌫</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btnDivide">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>÷</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnMultiple">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>×</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btnMinus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="btnPlus">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: rgb(255, 0, 127);
color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(85, 0, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="btnEqual">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: rgb(255, 255, 127);
	color: rgb(0, 0, 0);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(42, 42, 0);
}

QPushButton:pressed{
	background-color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLineEdit" name="display">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
